src/faction.cpp

```diff
@@ -339,7 +339,7 @@ void faction::randomize()
         } else {
             tries++;
         }
-    } while((one_in(num_values) || one_in(num_values)) && tries < 15);
+    } while( one_in( num_values ) && tries < 15 );
```

Comment:
This is not correct, one_in() effectively "rolls dice" every time you call it, so it is meaningful to say ```one_in( x ) || one_in( x )```

Reply:
If you feel like cleaning this up, the clearer option is to replace ```one_in( num_values ) || one_in( num_values )``` with ```x_in_y( 2, num_values )```